### DISCUSSION TITLE
[Feature Request]: Add Zoom in/out for Image in readme

### DISCUSSION THREAD

**Discussion by diamant3** [community/community#37623] [Mobile] [answered]
When it comes to images in Readme, they are so small to view, maybe you can add this feature, thanks.

**Comment by DenverCoder1** [comment, marked as answer]
If the images are wide enough in pixels, they can take up the full width of the section.

One way to allow users to make it bigger is by linking to original image (which I believe is the default). This allows them to see it full size in a new tab and zoom with their browser.

Additionally, if the image is displayed too small for the section, you may want to consider replacing the image with one where all text is more easily legible.

If none of the above works for you, I could agree it couldn't hurt to add this as an optional feature. Maybe by explicitly linking the image to the URL of the image could make an in-page zoom feature appear? This seems like it could be a neat idea.
